starter-code/scripts/articleView.js

```diff
@@ -53,13 +67,23 @@ articleView.setTeasers = function() {
   // Truncate logic to show only first two elements within the article body.
   $('.article-body *:nth-of-type(n+2)').hide();
   /* TODO: Add a delegated event handler to reveal the remaining paragraphs.
-    When a .read-on link is clicked, we can:
-    1. Prevent the default action of a link.
-    2. Reveal everything in that particular article now.
-    3. Hide that read-on link!
-
-    // STRETCH GOAl!: change the 'Read On' link to 'Show Less'
+  When a .read-on link is clicked, we can:
+  1. Prevent the default action of a link.
+  2. Reveal everything in that particular article now.
+  3. Hide that read-on link!
+  // STRETCH GOAl!: change the 'Read On' link to 'Show Less'
   */
+
+  $('article').on('click', '.read-on', function(){
+    event.preventDefault();
+    $(this).parent().find('*').show();
```

Comment:
Good work. Though the wild card selector does work, due be careful when using it. Normally you want to be specific to the element you are targeting.